### PR TITLE
Change more features from "experimental" to "under development"

### PR DESCRIPTION
**Pull request by prince-chrismc**
This should help make it more clear whats going into Conan 2.0

Needs #2868 before it can be reviewed

### CODE REVIEW

**Review by memsharded — APPROVED** [review]
Looking good!

**Comment by prince-chrismc** [comment]
I did a lot of the ones I knew easily but I am starting to hit my limited knowledge of the obscure features

image

There's about 100 left -- Maybe I can borrow someone from the team to help with the rest 🤔

File: extending/template_system/info_graph.rst [hunk 1]
```diff
@@ -6,9 +6,8 @@ Graph output for :command:`conan info` command
 
 .. warning::
 
-    This has to be considered as an **experimental** feature, we might
-    change the context provided to these templates once we have more examples
-    from the community.
+    This is a **migrated** feature. Please refer to :ref:`conan_v2_graph_info` learn about the changes.
+    Most of the functionality remains intact the main changes are to the command line.
```
Comment (RubenRBS) [inline extending/template_system/info_graph.rst:10]:
```suggestion
    Most of the functionality remains intact and the main changes are to the command line.
```

File: howtos/custom_generators.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@
 How to create and share a custom generator with generator packages
 ==================================================================
 
+.. warning::
+
+    This is a **deprecated** feature. Please refer to the :ref:`Migration Guidelines<conan2_migration_guide>`
+    to find the feature that replace this one.
```
Comment (RubenRBS) [inline howtos/custom_generators.rst:9]: For all of these, isn't an `s` (Or a `d`) missing for `replace`?

**Review by RubenRBS — APPROVED** [review]
Great work! These 2 are small grammar nitpicks

**Comment by prince-chrismc** [comment]
I'll need to fix that 🙊

**Pull request merged**